Documentation/frontera.rst

```diff
@@ -11,25 +11,23 @@ Add these lines to ``~/.bashrc``:
 
     module switch python3 python3/3.9.2
     module use /work2/09160/ulrich/frontera/spack/share/spack/modules/linux-centos7-cascadelake
-    module load seissol-env-develop-intel-19.1.1.217-x52n3zf
+    module load seissol-env
     export CC=mpiicc
     export CXX=mpiicpc
-    export FC=mpiifort
 
 This will load a preinstalled seissol-env module.
 
 Alternatively (and for reference), to compile seissol-env on Frontera, follow the procedure below:
 
 .. code-block:: bash
 
-    git clone --depth 1 --branch v0.18.1 https://github.com/spack/spack.git
+    git clone --depth 1 --branch v0.21.0 https://github.com/spack/spack.git
     cd spack
     echo "export SPACK_ROOT=$PWD" >> $HOME/.bashrc
     echo "export PATH=\$SPACK_ROOT/bin:\$PATH" >> $HOME/.bashrc
     # clone seissol-spack-aid and add the repository
-    git clone --branch supermuc_NG https://github.com/SeisSol/seissol-spack-aid.git
-    cd seissol-spack-aid
-    spack repo add ./spack
+    git clone https://github.com/SeisSol/seissol-spack-aid.git
+    spack repo add seissol-spack-aid/spack
     spack compiler find
 
 Following the workaround proposed in https://github.com/spack/spack/issues/10308, precise the module of the intel compilers in ``~/.spack/linux/compilers.yaml`` by changing ``modules: []`` to ``modules: ['intel/19.1.1']``.
```

Comment:
Is this still necessary with the change in line 14, same file?

Reply:
yes.